frontend/packages/kubevirt-plugin/src/k8s/wrapper/common/k8s-resource-wrapper.ts

```diff
@@ -4,7 +4,7 @@ import { apiVersionForModel, K8sKind, K8sResourceKind } from '@console/internal/
 import { Wrapper } from './wrapper';
 import { K8sResourceKindMethods } from '../types/types';
 
-export class K8sResourceWrapper<
+export abstract class K8sResourceWrapper<
```

Comment:
are we sure there is no use case of creating an instance of `K8sResourceWrapper`?

Reply:
It is better if the user creates its own definition if needed. Now it also requires model to be supplied.